### DISCUSSION TITLE
Unable to Open Website on 88.0.4314.0 Chromium Version using Taiko

### DISCUSSION THREAD

**Discussion by dhanushng88** [getgauge/taiko#2544] [General]
Whenever we use to run our scripts uisng the chromium browser we end up with a security page as attached in screenshot
![image](https://user-images.githubusercontent.com/52316009/157385282-47bba496-4175-414b-b873-2b8b7385a56c.png)

Even though we are handling the security contexts we are unable to pass this page
beforeScenario(async () => {
    if (process.env.api_test_run === 'false') {
        await setConfig({ navigationTimeout: parseInt(process.env.test_timeout) });

        if (process.env.host === "local") {
            await openBrowser({ headless: headlessChrome, ignoreCertificateErrors: true, args: ['--no-sandbox', '--start-fullscreen', '--disable-web-security', '--disable-RendererCodeIntegrity'] })
            await deleteCookies();
        } else if (process.env.host === "docker") {
            await openBrowser({ headless_chrome: true, ignoreCertificateErrors: true, args: ['--disable-setuid-sandbox', '--no-sandbox', '--start-fullscreen', '--disable-web-security'] })
            await deleteCookies();
        }
    }
});

NOTE: Its working fine on Chromium Browser version 70 and above but seeing issues with 80 and above

**Comment by dhanushng88** [comment]
@NivedhaSenthil @zabil Any idea how to resolve this??

**Comment by zabil** [comment]
There are some pointers at https://support.google.com/chrome/thread/88331714/error-the-information-you%E2%80%99re-about-to-submit-is-not-secure?hl=en

**Reply by dhanushng88** [reply]
@zabil After updating the Taiko version and Chromium version its working fine now.. Thanks for your inputs
